woocommerce/woocommerce-ios
# [IAP] Update Site's storage layer with `isSiteOwner` property

Branched from https://github.com/woocommerce/woocommerce-ios/pull/9963 , which needs to be merged first.
Most of the 900+ lines come from the data model generation.


## Description
This PR updates the Core Data's `Site` entity with the `isSiteOwner` property, which will be used to add logic when the user is/is not the site's owner. This is helpful when we have multiple administrators, and specifically for In-App Purchases since only the owner can manage purchases.

## Testing instructions
Build the app and make sure that there's no crash. We're not using this property yet anywhere, tests should pass.
